src/mono/sample/iOS-NativeAOT/Makefile

```diff
@@ -6,7 +6,7 @@ BUILD_CONFIG?=Debug
 TARGET_ARCH?=$(shell . $(TOP)eng/native/init-os-and-arch.sh && echo $${arch})
 TARGET_OS?=iossimulator
 DEPLOY_AND_RUN?=false
-STRIP_DEBUG_SYMBOLS?=false
+STRIP_DEBUG_SYMBOLS?=true
```

Comment:
Same comment as for MonoAOT: https://github.com/dotnet/runtime/pull/85317/files#diff-8f074c3f5f920c745799be4983690457f3b036bc566a65d5644b95568ba4b34dL9